app/src/features/surveys/view/SurveyHeader.tsx

```diff
@@ -191,6 +191,17 @@ const SurveyHeader = () => {
                 </Box>
               </Box>
               <Box display="flex" alignItems="flex-start" flex="0 0 auto" className={classes.pageTitleActions}>
+                <Button
+                  title="Submit Survey Data and Documents"
+                  color="primary"
+                  variant="contained"
+                  onClick={() => history.push('observations')}
+                  sx={{
+                    minWidth: '7rem',
+                    marginRight: '0.5em'
```

Comment:
Can we use MUI units instead of ems? e.g. `sx={{ minWidth: '7rem', mr: 1 }}`

Reply:
done